shell/platform/embedder/embedder_layers.cc

```diff
@@ -180,10 +180,10 @@ void EmbedderLayers::PushPlatformViewLayer(
   layer.platform_view = platform_views_referenced_.back().get();
 
   const auto layer_bounds =
-      SkRect::MakeXYWH(params.offsetPixels.x(),                          //
-                       params.offsetPixels.y(),                          //
-                       params.sizePoints.width() * device_pixel_ratio_,  //
-                       params.sizePoints.height() * device_pixel_ratio_  //
+      SkRect::MakeXYWH(params.offsetPixels().x(),                          //
```

Comment:
is this just `finalBoundingRect()` now?

Reply:
I think the size is not the same as the bounding rect. The size points are the original size before applying mutation.

Reply:
@chinmaygarde I thought params.sizePoints did not consider scaling and rotation at all? Does embedder free from scaling and rotation layers?

Reply:
I believe the embedder is supposed to apply the scaling and rotation themselves from the data present in the mutator stack to get the final width and height. I believe there is a test for this in the embedder unit-tests harness and this should be documented in the header.

Reply:
I meant this rect returned from this method does not include rotation and scaling. Want to make sure if the original size used here is intended.

Reply:
Yes. It is.